front/tests/pages/op-simulation-settings-page-model.ts

```diff
@@ -0,0 +1,47 @@
+import { type Locator, type Page, expect } from '@playwright/test';
+
+class OperationalStudiesSimulationSettingsPage {
+  readonly page: Page;
+
+  readonly electricalProfilesSwitch: Locator;
+
+  readonly linearMarginSwitch: Locator;
+
+  readonly marecoMarginSwitch: Locator;
+
+  readonly codeCompoSelector: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.electricalProfilesSwitch = page.locator('#usingElectricalProfiles');
+    this.linearMarginSwitch = page.locator('#constraint-distribution-switchSTANDARD + label');
+    this.marecoMarginSwitch = page.locator('#constraint-distribution-switchMARECO + label');
+    this.codeCompoSelector = page.locator('#speed-limit-by-tag-selector');
+  }
+
+  // Activate electrical profiles switch
+  async activateElectricalProfile() {
```

Comment:
How does it work if it's already checked ?

Reply:
Changed the function to `checkElectricalProfile()`